src/Stryker.ts

```diff
@@ -102,6 +104,13 @@ export default class Stryker {
     });
   }
 
+  private freezeConfig() {
+    freezeRecursively(this.config);
+    if (log.isDebugEnabled()) {
```

Comment:
Yes correct. Its a minor thing but sometimes it might be usefull
Op 22 apr. 2016 08:55 schreef "Simon de Lang" <notifications@github.com>:

In src/Stryker.ts
<https://github.com/stryker-mutator/stryker/pull/94#discussion_r60697760>:

> @@ -102,6 +104,13 @@ export default class Stryker {
>      });
>    }
>
> +  private freezeConfig() {
> +    freezeRecursively(this.config);
> +    if (log.isDebugEnabled()) {

Is this check added for performance reasons as you don't want to spend time
stringifying the config the user does not even see the log message?

—
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub
<https://github.com/stryker-mutator/stryker/pull/94/files/c6d9d2b7e42d56de0419cc0b75d55379b38047a5#r60697760>